app/design/adminhtml/default/default/template/login.phtml

```diff
@@ -35,26 +33,96 @@
+                        <div class="clear"></div>
+                        <div class="form-buttons">
+                            <a class="left" href="<?= Mage::helper('adminhtml')->getUrl('adminhtml/index/forgotpassword', ['_nosecret' => true]) ?>"><?= Mage::helper('adminhtml')->__('Forgot your password?') ?></a>
+                            <input type="submit" class="form-button" value="<?= Mage::helper('core')->quoteEscape(Mage::helper('adminhtml')->__('Login')) ?>" title="<?= Mage::helper('core')->quoteEscape(Mage::helper('adminhtml')->__('Login')) ?>" />
+                        </div>
                     </div>
-                    <div class="input-box input-right">
-                        <label for="login"><?= Mage::helper('adminhtml')->__('Password:') ?></label><br />
-                        <input type="password" id="login" name="login[password]" value="" class="required-entry input-text" />
-                    </div>
-                    <?= $this->getChildHtml('form.additional.info') ?>
-                    <div class="clear"></div>
-                    <div class="form-buttons">
-                        <a class="left" href="<?= Mage::helper('adminhtml')->getUrl('adminhtml/index/forgotpassword', ['_nosecret' => true]) ?>"><?= Mage::helper('adminhtml')->__('Forgot your password?') ?></a>
-                        <input type="submit" class="form-button" value="<?= Mage::helper('core')->quoteEscape(Mage::helper('adminhtml')->__('Login')) ?>" title="<?= Mage::helper('core')->quoteEscape(Mage::helper('adminhtml')->__('Login')) ?>" />
+                    <div id="step2" class="no-display">
+                        <div class="input-box input-left">
+                            <label for="twofa_verification_code"><?= Mage::helper('adminhtml')->__('2FA verification code:') ?></label><br />
+                            <input type="text" id="twofa_verification_code" name="login[twofa_verification_code]" value="" class="required-entry input-text validate-number" autocomplete="one-time-code" disabled />
+                        </div>
+                        <div class="clear"></div>
+                        <div class="form-buttons">
+                            <a id="back-button" class="left" href="#"><?= Mage::helper('adminhtml')->__('Back') ?></a>
+                            <input type="submit" class="form-button" value="<?= Mage::helper('core')->quoteEscape(Mage::helper('adminhtml')->__('Login')) ?>" title="<?= Mage::helper('core')->quoteEscape(Mage::helper('adminhtml')->__('Login')) ?>" />
+                        </div>
                     </div>
                 </div>
                 <p class="legal">Powered by Maho</p>
             </form>
             <div class="bottom"></div>
-            <script type="text/javascript">
-                var loginForm = new varienForm('loginForm');
+            <script type="module">
+                history.replaceState(null, null, window.location.href);
```

Comment:
this way?

Reply:
ya, it seems to be working for me. Now on a refresh it's a regular GET instead of POST.